habitat-lab/habitat/articulated_agents/humanoids/kinematic_humanoid.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import List
+
+import magnum as mn
+import numpy as np
+
+import habitat_sim.physics as phy
+from habitat.articulated_agents.mobile_manipulator import (
+    ArticulatedAgentCameraParams,
+    MobileManipulator,
+    MobileManipulatorParams,
+)
+
+
+class KinematicHumanoid(MobileManipulator):
+    def _get_humanoid_params(self):
+        return MobileManipulatorParams(
+            arm_joints=[],  # For now we do not add arm_joints
+            gripper_joints=[],
+            wheel_joints=None,
+            arm_init_params=None,
+            gripper_init_params=None,
+            gripper_closed_state=np.array([]),
+            gripper_open_state=np.array([]),
+            gripper_state_eps=None,
+            wheel_mtr_pos_gain=None,
+            wheel_mtr_vel_gain=None,
+            wheel_mtr_max_impulse=None,
+            ee_offset=[mn.Vector3(), mn.Vector3()],
+            ee_links=[15, 9],
+            ee_constraint=np.zeros((2, 2, 3)),
+            cameras={
+                "robot_head": ArticulatedAgentCameraParams(
```

Comment:
Thank you for working on this! Do we need another name like "human_head" instead of "robot_head"?

Reply:
Good point. I think the best would be to replace all these to articulated_agent_agent_head, so that we can get the observations without much code duplication. Maybe this should be in a separate PR?

Reply:
I agree a separate PR would help us move fast on this. Please make a note this will need to be changed later.

Reply:
Note taken